templates/common/infra/terraform/core/host/appservice/appservicepython/appservicepython_variables.tf

```diff
@@ -0,0 +1,51 @@
+variable "location" {
+  description = "The supported azure location where the resource deployed"
+  type        = string
+}
+
+variable "rg_name" {
+  description = "The name of the resource group to deploy resources into"
+  type        = string
+}
+
+variable "appservice_plan_id" {
+  description = "The id of the appservice plan to use."
+  type        = string
+}
+
+variable "service_name" {
+  description = "A name to reflect the type of the app service e.g: web, api."
+  type        = string
+}
+
+variable "app_settings" {
+  description = "A list of app settings pairs to be assigned to the app service"
+  type        = map(string)
+}
+
+variable "identity" {
+  description = "A list of application identity"
+  type        = list(any)
+  default     = []
+}
+
+variable "app_command_line" {
+  description = "The cmd line to configure the app to run."
+  type        = string
+}
+
+variable "tags" {
+  description = "A list of tags used for deployed services."
+  type        = map(string)
+}
+
+variable "resource_token" {

```

Comment:
Isn't resource_token a global var?  DOes it need to be an input param?  I don't have it as an input param in bicep to clean up the module interface.

Reply:
it was available to all when all files used to be in the same folder. now it needs to be passed as input variable to the reusable modules  as they have their own folder under /modules